cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/GoalOptimizer.java

```diff
@@ -113,7 +113,7 @@ public GoalOptimizer(KafkaCruiseControlConfig config,
     _progressUpdateLock = new AtomicBoolean(false);
     // A new AtomicReference with null initial value.
     _proposalGenerationException = new AtomicReference<>();
-    _proposalPrecomputingProgress = new OperationProgress();
+    _proposalPrecomputingProgress = new OperationProgress("");
```

Comment:
didn't you explicitly overload OpeartionProgress constructor so that no arguments will still work?